src/Razor/src/Microsoft.VisualStudio.LanguageServerClient.Razor/Endpoints/Completion.cs

```diff
@@ -309,6 +309,11 @@ private static TextEdit BuildRevertedEdit(TextEdit provisionalTextEdit)
 
     private void AddSnippetCompletions(DelegatedCompletionParams request, ref PooledArrayBuilder<CompletionItem> builder)
     {
+        if (!request.ShouldIncludeSnippets)
```

Comment:
One other option we have here is to make this `ShouldIncludeBuiltInSnippets`. I'm not too opinionated here